config-example.yaml

```diff
@@ -95,6 +95,16 @@ derp:
     # For more details on how this works, check this great article: https://tailscale.com/blog/how-tailscale-works/
     stun_listen_addr: "0.0.0.0:3478"
 
+    # For better connection stability (especially when using an Exit-Node and DNS is not working), 
+    # it is possible to optionall add the public IPv4 and IPv6 address to the Derp-Map using:
+    IPv4: 1.2.3.4
+    IPv6: 2001:db8::1
+
+    # This flag can be used, so the DERP map entry for the embedded DERP server is not written automatically,
+    # it enables the creation of your very own DERP map entry using a locally available file with the parameter DERP.paths
+    # If you enable the DERP server and set this to true, it is required to add the DERP server to the DERP map using DERP.paths
+    manual_derp_map: false
```

Comment:
I find this name a bit hard to get, I would guess that the majority of people would not use this option, so it could have a long and verbose name:

`disable_inject_of_private_derp_map_entry`: false

but double negative is hard, so we could have enabled by default:

`add_private_derp_map_entry`: true

Reply:
I'm not particularly convinced by any of my suggestions either, so open to suggestions.

Reply:
Since we are talking about the "embedded DERP server" in the config, what about:
`automatically_add_embedded_derp_region: true`

Default should then be changed to `true` obviously and if set `false` it needs to be defined using the `paths` variable.

Reply:
I think that sounds reasonable.

Reply:
Hi @kradalby I added your requested change in 46d28909893ef18f1be38d99baec4a5ecfdae617